py-polars/polars/internals/io.py

```diff
@@ -250,3 +273,145 @@ def _update_columns(df: pli.DataFrame, new_columns: list[str]) -> pli.DataFrame:
         new_columns = cols
     df.columns = new_columns
     return df
+
+
+def _xl_setup_workbook(
```

Comment:
Can we make a separate `io_excel` module all excel related code? This is becoming more than a wrapper to our rust backends. That will keep the logic separated.

Reply:
Makes sense; will take care of it ;)